src/SFA.DAS.EAS.Web/Controllers/TransfersController.cs

```diff
@@ -23,8 +23,8 @@ public class TransfersController : BaseEmployerController
 
         [HttpGet]
         [OutputCache(CacheProfile = "NoCache")]
-        [Route("{hashedCommitmentId}/approve")]
-        public async Task<ActionResult> TransferApproval(string hashedAccountId, string hashedCommitmentId)
+        [Route("{hashedCommitmentId}")]
```

Comment:
One slight issue is that once approved or rejected, the user cant reload the page or come back to this confirmation page without a generic error.